modules/nw-ovn-kubernetes-list-database-contents.adoc

```diff
@@ -0,0 +1,218 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="nw-list-ovn-northbound-database-contents_{context}"]
+= Listing OVN northbound database contents with ovn-kubernetes
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+* The OpenShift CLI (oc) installed.
+
+To understand logic flow rules you need to examine the northbound database and understand what objects are there to see how they are translated into logic flow rules.
+
+Follow this procedure to list OVN database contents with OVN-Kubernetes.
+
+.Procedure
+
+. Find the OVN raft leader for the northbound database.
```

Comment:
We already showed how to find the raft leader in the "Querying the status of the northbound and southbound databases" section.  I don't think we need to repeat it here.

Reply:
@CFields651 I guess I feel maybe user will not read doc in linear fashion and might just see this section in TOC so they will simply go to this section, so I was keeping it all self-contained. Personally if I go to a section I like for the most part having everything I need there within reason

Reply:
@CFields651 I want to keep every section self-contained, I assume customers will not read linearly